README.md

```diff
@@ -116,7 +116,7 @@ Container images are configured using parameters passed at runtime (such as thos
 | `-e PUID=1000` | for UserID - see below for explanation |
 | `-e PGID=1000` | for GroupID - see below for explanation |
 | `-e TZ=America/New_York` | Specify a timezone to use EG America/New_York |
-| `-e REDIS_URL=` | Specify an external redis instance to use. Can optionally include a port (`redis:6379`) and/or db (`redis/foo`). If left blank or not included, will use a built-in redis instance. If changed after initial setup will also require manual modification of /config/settings.py |
```

Comment:
This is not how to edit the readme. You need to change the readme-vars.yml

Reply:
doh! should have read the top of the readme - I just did ctrl+f for redis. fixed now.